src/eager/ort_eager.cpp

```diff
@@ -2,12 +2,45 @@
 // Licensed under the MIT License.
 
 #include <torch/extension.h>
-
+#include "torch/csrc/autograd/python_variable.h"
 #include "ort_backends.h"
 #include "ort_log.h"
+#include "ort_aten.h"
+#include "orttraining/core/framework/ortmodule_graph_builder.h"
+#include "python/onnxruntime_pybind_state_common.h"
+#include "orttraining/core/framework/torch/dlpack_python.h"
+
+namespace onnxruntime{
+namespace python{
+  onnxruntime::Environment& GetEnv();
+  void addGlobalMethods(py::module& m, Environment& env);
+  void addObjectMethods(py::module& m, Environment& env);
+  void addOrtValueMethods(pybind11::module& m);
+  void addIoBindingMethods(pybind11::module& m);
+  void addObjectMethodsForTraining(py::module& m);
+}
+}
 
 namespace torch_ort {
 namespace eager {
+using namespace onnxruntime::training;
+
+py::object ORTTensor_toDLPack(const at::Tensor& data)
+{
+  //const at::Tensor& ort_at_tensor = THPVariable_Unpack(data);
+  OrtValue ort_value = create_ort_value(data);
+  return py::reinterpret_steal<py::object>(onnxruntime::training::framework::torch::ToDlpack(ort_value));
```

Comment:
Given that you have a using namespace onnxruntime::training consider shortening to framework::torch::ToDlpack